Task_TheGridProblem/index.html

```diff
@@ -1,11 +1,23 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Grid Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for GRID PROBLEM TASK-->
-  </body>
```

Comment:
Consider using more descriptive class names for the grid items instead of grid-item1, grid-item2, etc.